src/components/Calendar/Calendar.tsx

```diff
@@ -96,7 +109,7 @@ const Calendar: React.FC<CalendarProps> = ({
                         center: "title",
                         right: "dayGridMonth,timeGridWeek,timeGridDay",
                     }}
-                    events={initialEvents}
+                    events={displayEvents.current}
```

Comment:
what is `.current` - is this a `useRef` thing?

Reply:
yes